docs/source/documentation.rst

```diff
@@ -153,14 +85,14 @@ This integrity test automatically performs the axon and myelin segmentation of a
 Comprehensive test
 ~~~~~~~~~~~~~~~~~~
 
-To run the entire testing suite (more code coverage), go to your AxonDeepSeg project directory on the terminal and run ``py.test``::
+To run the entire testing suite (more code coverage), go to your AxonDeepSeg project directory on the terminal, activate the virtual environment, and then run ``py.test``::
 
     cd axondeepseg
+    ads_activate
     py.test --cov AxonDeepSeg/ --cov-report term-missing
 
 If all tests pass, AxonDeepSeg was installed succesfully.
```

Comment:
This is probably out of scope for this PR, but something to note: SCT has a dedicated CLI script for exposing `pytest` called `sct_testing`.

Maybe ADS could do something similar? For example, in `axondeepseg_test`, you could add a `--full` option that will run all of the the pytest tests. That way, it isn't necessary to `cd` and activate the environment?

Reply:
Interesting - I'll open an issue and we'll get that done later!

Reply:
https://github.com/axondeepseg/axondeepseg/issues/822